mm2src/mm2_main/src/lp_swap/maker_swap_v2.rs

```diff
@@ -194,24 +246,52 @@ impl<MakerCoin: Send + 'static, TakerCoin: Send + 'static> InitialState for Init
 }
 
 #[async_trait]
-impl<MakerCoin: MmCoin + SwapOpsV2 + Send + Sync + 'static, TakerCoin: MmCoin + SwapOpsV2 + Send + Sync + 'static> State
-    for Initialize<MakerCoin, TakerCoin>
-{
+impl<MakerCoin: MmCoin + CoinAssocTypes, TakerCoin: MmCoin + SwapOpsV2> State for Initialize<MakerCoin, TakerCoin> {
     type StateMachine = MakerSwapStateMachine<MakerCoin, TakerCoin>;
 
     async fn on_changed(self: Box<Self>, state_machine: &mut Self::StateMachine) -> StateResult<Self::StateMachine> {
+        {
+            let sql_params = params![
```

Comment:
I think using `owned_named_params` macro https://github.com/KomodoPlatform/komodo-defi-framework/blob/2789a1bc792e54c17cd7fee5b3269cdd8990fa60/mm2src/db_common/src/sqlite.rs#L17-L26 can make it easier to avoid mistakes in the order of params.

Reply:
Good note, `named_params` turned out to be sufficient here, thanks!